src/frontend/src/test_utils.rs

```diff
@@ -446,13 +450,21 @@ impl UserInfoWriter for MockUserInfoWriter {
 impl MockUserInfoWriter {
     pub fn new(user_info: Arc<RwLock<UserInfoManager>>) -> Self {
         user_info.write().create_user(UserInfo {
+            id: DEFAULT_SUPPER_USER_ID,
             name: DEFAULT_SUPPER_USER.to_string(),
             is_supper: true,
             can_create_db: true,
             can_login: true,
             ..Default::default()
         });
-        Self { user_info }
+        Self {
+            user_info,
+            id: AtomicU32::new(1 + RESERVED_USER_ID as u32),
```

Comment:
I see two places using `1 + RESERVED_USER_ID` and no places using `RESERVED_USER_ID`, and a future potential caller may forget to `+1`. Is it better to define sth like `NONRESERVED_USER_ID` instead?